integration/mesos/src/main/java/tachyon/mesos/TachyonFramework.java

```diff
@@ -242,6 +247,8 @@ public void statusUpdate(SchedulerDriver driver, Protos.TaskStatus status) {
         case TASK_ERROR:
           if (status.getTaskId().getValue().equals(String.valueOf(mMasterTaskId))) {
             mMasterCount --;
+          } else if (status.getTaskId().getValue().equals(String.valueOf(mWorkerTaskId))) {
+            mWorkerCount --;
```

Comment:
This means that a worker or master task is lost or failed. I will add some message